src/rai_tts/rai_tts/tts_node.py

```diff
@@ -53,7 +53,7 @@ def __init__(self):
         self.job_id: int = 0
         self.queued_job_id = 0
         self.tts_client = self._initialize_client()
-        self.create_timer(0.01, self.status_callback)
+        self.create_timer(0.25, self.status_callback)
```

Comment:
How about declaring this as a `ros2 parameter`? 
It's hard to know the meaning of this param at the first glance